### ISSUE TITLE
Image type generated by Chart.getImage not specified in the documentation

### ISSUE THREAD

**Issue by reego-fr** [OfficeDev/office-js-docs#709]
Article: office-js-docs / reference / excel / chart.md 
Issue: 
Hi,
I am experiencing issues with the Chart.getImage method. I am getting inconsistent results when switching between "filling modes". It is the kind of issue that should be raised via uservoice, right ?

In the meantime, regarding the documentation, the first thing I though when reading it was : what kind of image gets generated ?
The current description is pretty vague : Renders the chart as a base64-encoded image by scaling the chart to fit the specified dimensions. 
I would rephrase it that way : Renders the chart as a base64-encoded PNG image by scaling the chart to fit the specified dimensions. 

What do you think about it ?

**Comment by mlafleur** [comment]
For general questions/issues you're finding (like inconsistent results) I would recommend [Stack Overflow](http://stackoverflow.com/questions/tagged/office-js). SO is monitored by both internal Microsoft and overall community experts. Just be sure and tag it with `office-js` so it surfaces with the right audience.

**Comment by reego-fr** [comment]
Thank you for your answer @mlafleur 
I already sent a feedback on uservoice and I now have more details to help one reproduce the issue, high DPI settings happens to trigger the issue, see https://officespdev.uservoice.com/forums/224641-feature-requests-and-feedback/suggestions/17752873-fix-chart-getimage-output for more details. 
I wonder if it is worth sharing on stack overflow too, I might have to refactor a bit the code I am using to make it more readable and be able to share it.

**Comment by mlafleur** [comment]
Could you post that issue to Stack Overflow? The designers of UserVoice assumed each post is a Feature Request that votes and comments are then attached too. This doesn't support the back-and-forth typically required in sorting out unexpected behaviors. Also, your posting won't been seen right away as planning features and roadmap doesn't happen day-to-day). 

Posting this on Stack Overflow will ensure it is seen by a broad community of engineers both inside and outside of Microsoft.  Having read what you've written, I have some thoughts to add once it is on SO.

**Comment by reego-fr** [comment]
Thank you for your feedback, I will post it on SO too. I will only be able to do it by tomorrow I guess, see you there :-)
regards,

**Comment by reego-fr** [comment]
For you information, here is the post I created on SO : http://stackoverflow.com/questions/41739642/how-to-get-an-image-of-an-excel-chart-out-of-javascript-api-chart-getimage
